selenium_tests/conftest.py

```diff
@@ -0,0 +1,161 @@
+import pytest
+import os
+from unittest.mock import patch
+from types import SimpleNamespace
+
+from django.db import connection
+from django.conf import settings
+from selenium.webdriver import (
+    DesiredCapabilities,
+    Remote,
+)
+
+from selenium_tests.util import (
+    Browser,
+    DEFAULT_PASSWORD,
+    DatabaseLoader,
+    kill_connection_sql
+)
+from selenium_tests.data_util import create_user
+from selenium_tests.page import LoginPage
+from courses.factories import (
+    ProgramFactory,
+    CourseRunFactory,
+)
+from dashboard.models import ProgramEnrollment
+from financialaid.models import (
+    Tier,
+    TierProgram,
+)
+from search.indexing_api import (
+    delete_index,
+    recreate_index,
+)
+from roles.roles import Staff
+from roles.models import Role
+
+
+@pytest.fixture(scope='session')
+def django_db_setup(django_db_setup, django_db_blocker):
+    with django_db_blocker.unblock():
+        with connection.cursor() as cur:
+            cur.execute('DROP TABLE IF EXISTS wagtailsearch_editorspick CASCADE;')
+            DatabaseLoader().create_backup(db_cursor=cur)
+
+
+@pytest.fixture(autouse=True)
+def _db_loader(request, django_db_blocker):
+    marker = request.keywords.get('django_db', None)
+    if marker:
+        database_loader = DatabaseLoader()
+        with django_db_blocker.unblock():
+            with connection.cursor() as cur:
+                cur.execute(kill_connection_sql)
+            connection.close()
+        database_loader.load_backup()
+
+
+@pytest.fixture(scope='session')
+def internal_api_patcher():
```

Comment:
This was changed recently, we can just do the patches local to the test code as long as we're careful about waiting for the server to execute before we exit the patch